app/graphql/types/payloads/update_comment_payload.rb

```diff
@@ -0,0 +1,8 @@
+module Types
+  module Payloads
+    class UpdateCommentPayload < Types::BaseObject
+      field :comment, CommentType, null: true
```

Comment:
In your realisation it should be:
```suggestion
      field :comment, CommentType, null: false
```